test/srb-en.html

```diff
@@ -0,0 +1,48 @@
+---
+title: Basic search Page using Headless
+description: Demo page for the Canada.ca Search UI Basic
+lang: en
+altLangPage: srb-fr.html
+nositesearch: true
+dateModified: 2024-03-14
+stylesheets:
+    - href: "../src/connector.css"
+---
+
+<!--FORM-->
+<form method="GET" action="#wb-land" role="form" class="mrgn-tp-sm">
+	<div class="input-group mrgn-tp-lg">
+	<label class="wb-inv" for="sch-inp-ac">Search Government of Canada websites</label>
+		<input id="sch-inp-ac" class="form-control" value="" name="q" dir="ltr" list="datalist-sch-inp-ac" autocomplete="off" spellcheck="false" type="search" data-fusion-query="safe" aria-describedby="gc-pi">
+		<span class="input-group-btn">
+		<button id="sch-inp" class="btn btn-primary btn-small" name="wb-srch-sub" type="submit"> <span class="glyphicon-search glyphicon"></span> <span class="wb-inv">Search</span> </button>
+		</span> </div>
+	<p id="gc-pi" class="mrgn-tp-md">Don’t include personal information (telephone, email, SIN, financial, medical, or work details).</p>
+</form>
+
+<!--RESULTS-->
+<div data-analytics-id="squery">
+	<div class="page-type-search" data-srchprms='{
```

Comment:
We could rename it but it has lots of default values so it really is to overwrite. But yes, we could change it so anything.

For the security aspect, it is not a security issue but I'm sure it could be better tweaked for the purpose of this GitHub repo later on. On our CMS, we have a different approach with this.